### PR TITLE
lots of fixes

### LINKED ISSUE #18: defaults question
Do we actually want to default base_registery_username and base_registery_password to ee_registry_username and ee_registry_password? I am not sure when/if this would be the "normal" case.

https://github.com/djdanielsson/ee_utilities/blob/f6532aa752f2af84639c8314234afab179e92abb/roles/ee_builder/defaults/main.yml#L13

### PR DESCRIPTION
**Pull request by djdanielsson**
### What does this PR do?
fixed a lot of spelling issues, renamed some variables, swaps some defaults around

### How should this be tested?
Automated might work, I have tested this against my own code but I can't say it worked before this since these changes will be breaking old code.

### Is there a relevant Issue open for this?
Provide a link to any open issues that describe the problem you are solving.
resolves #18 

### Other Relevant info, PRs, etc
none

### CODE REVIEW

**Comment by sean-m-sullivan** [comment]
@anshulbehl hoping this works, we need to get you in the COP so I can ping you.

**Review by Tompage1994 — APPROVED** [review]
LGTM

File: roles/ee_builder/defaults/main.yml [hunk 1]
```diff
@@ -4,13 +4,13 @@ ee_container_runtime: podman
...
 galaxy_cli_opts: '-v'
 
-ee_stream: downstream
+ee_stream: "{% if ee_base_registry_username is defined %}downstream{% else %}upstream{% endif %}"
 
 # Image defaults
-ee_base_image: "{{ __execution_environment_defintion.base_image | default( __ee_stream_images[ee_stream].base_image ) }}"
+ee_base_image: "{{ __execution_environment_definition.base_image | default( __ee_stream_images[ee_stream].base_image ) }}"
 ee_builder_image: "{{ __ee_stream_images[ee_stream].builder_image }}"
-base_registery_username: "{{ ee_registry_username | default(omit, true) }}"
```
Comment (sean-m-sullivan) [inline roles/ee_builder/defaults/main.yml:12]: with this change, we need a changelog entry file.

**Review by sean-m-sullivan — CHANGES_REQUESTED** [review]
LGTM other then the one change to add a changelog entry.

File: roles/ee_builder/defaults/main.yml (hunk 1 above)
Comment (djdanielsson) [reply roles/ee_builder/defaults/main.yml:12]: am I blind? I don't see the change log stuff. @sean-m-sullivan

**Review by anshulbehl — APPROVED** [review]

**Pull request merged**